wg-batch/charter.md

```diff
@@ -0,0 +1,92 @@
+# WG Batch Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [wg-governance].
+
+[Kubernetes Charter README]: /committee-steering/governance/README.md
+
+## Scope
+
+Discuss and enhance the support for Batch (eg. HPC, AI/ML, data analytics, CI)
+workloads in core Kubernetes. We want to unify the way users deploy batch
+workloads to improve portability and to simplify supportability for Kubernetes
+providers.
+
+### In scope
+
+- The following enhancements:
+  - Additions to the batch API group, currently including Job and CronJob resources
+    that benefit batch use cases such as HPC, AI/ML, data analytics and CI.
+  - Primitives for job-level queueing, not limited to the k8s Job resource. Long-term,
+    this could include multi-cluster support.
+  - Primitives to control and maximize utilization of resources in fixed-size clusters (on-prem) and elastic clusters (cloud).
+  - Runtime and scheduling support for specialized hardware (GPUs, NUMA, RDMA, etc.)
+- To congregate leads and users from different external projects and user groups
+  (CNCF TAGs, conferences and workshops) in the batch ecosystem to gather requirements,
+  validate designs and encourage reutilization of core kubernetes APIs.
+
+### Out of scope
+
+- Enhancements with the following nature:
+  - Addition of new API kinds that serve a specialized type of workload. The focus
+    should be on general APIs that specialized controllers can build on top of.
+  - Uses of the batch APIs as support for serving workloads (eg. backups,
+    upgrades, migrations). These can be served by existing SIGs.
+  - Proposals that duplicate the functionality of core kubernetes components
+    (job-controller, kube-scheduler, cluster-autoscaler).
+  - Job workflows or pipelines. Mature third party frameworks serve these
+    use cases with the current kubernetes primitives. But additional primitives
+    to support these frameworks could be in scope.
+- To dictate a _one-true-way_ to support batch workloads in Kubernetes.
```

Comment:
k8s by design is extensible, so I am not sure how to evaluate what we propose against this criteria. Any new API we propose could be seen as "dictating" something.

@liggitt what is the suggestion from steering committee here to avoid exactly?